juriscraper/opinions/united_states/administrative_agency/bia.py

```diff
@@ -118,7 +118,7 @@ def extract_from_text(self, scraped_text: str) -> Dict[str, Any]:
         date = re.findall(
             r"Decided (by (Acting\s)?Attorney General )?(.*\d{4})",
             scraped_text,
-        )[0][1]
+        )[0][-1]
```

Comment:
Um? This is a bit sus, no?

Reply:
suspect?  ... 

Making the test let me find a bug in my regex pattern.  I was trying to sneak in the fix for BIA

Reply:
Sus == suspect. Gotta hang with the kids, Bill!

Fix makes sense. Glad to see the new test working!

Reply:
oh dont worry my wife brings home the lingo from the kids